+++
title = "High‐throughput monitoring of wild bee diversity and abundance via mitogenomics"
date = "2015-07-15"
authors = ["Tang M", "Hardman CJ", "Ji Y", "Meng G", "Liu S", "Tan M", "Yang S", "Moss ED", "Wang J", "Yang C", "Bruce C"]
publication_types = ["2"]
publication = "Methods in Ecology and Evolution, https://doi.org/10.1111/2041-210X.12416"
publication_short = "Methods in Ecology and Evolution, https://doi.org/10.1111/2041-210X.12416"
abstract = "Summary

1. Bee populations and other pollinators face multiple, synergistically acting threats, which have led to population declines, loss of local species richness and pollination services, and extinctions. However, our understanding of the degree, distribution and causes of declines is patchy, in part due to inadequate monitoring systems, with the challenge of taxonomic identification posing a major logistical barrier. Pollinator conservation would benefit from a high‐throughput identification pipeline.

2. We show that the metagenomic mining and resequencing of mitochondrial genomes (mitogenomics) can be applied successfully to bulk samples of wild bees. We assembled the mitogenomes of 48 UK bee species and then shotgun‐sequenced total DNA extracted from 204 whole bees that had been collected in 10 pan‐trap samples from farms in England and been identified morphologically to 33 species. Each sample data set was mapped against the 48 reference mitogenomes.

3. The morphological and mitogenomic data sets were highly congruent. Out of 63 total species detections in the morphological data set, the mitogenomic data set made 59 correct detections (93·7% detection rate) and detected six more species (putative false positives). Direct inspection and an analysis with species‐specific primers suggested that these putative false positives were most likely due to incorrect morphological ID s. Read frequency significantly predicted species biomass frequency (R 2 = 24·9%). Species lists, biomass frequencies, extrapolated species richness and community structure were recovered with less error than in a metabarcoding pipeline.

4. Mitogenomics automates the onerous task of taxonomic identification, even for cryptic species, allowing the tracking of changes in species richness and distributions. A mitogenomic pipeline should thus be able to contain costs, maintain consistently high‐quality data over long time series, incorporate retrospective taxonomic revisions and provide an auditable evidence trail. Mitogenomic data sets also provide estimates of species counts within samples and thus have potential for tracking population trajectories."
abstract_short = ""
image_preview = ""
selected = false
projects = []
tags = []
url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""
math = true
highlight = true
[header]
image = ""
caption = ""
+++