# Multiplex sequencing of pooled mitochondrial genomes

The main motivation for multiplex sequencing and reconstructing mitogenomes from pooled taxa is to reduce analytical cost on individual library construction required for HTS. In principle, the more taxa that can be pool-sequenced, the less the average cost for each species, to the point where the main cost per taxon is mainly determined by its sequencing volume and the associated computational cost. In practice, a number of factors must be balanced: total number of pooled taxa, phylogenetic distance among taxa, DNA quality and quantity, and total sequencing volume. Empirical analysis will also need to consider specific features associated to the employed sequencing technology and assembly programs. In this study, we seek to answer these questions and develop a new pipeline for rapid and accurate reconstruction of multiplex mitogenomes from pooled taxa without relying on any DNA enrichment or amplification. In addition, we explore the plausibility of a multi-locus identification approach that integrates full mitogenome sequences or ‘mito-metagenomics’. (see more on https://doi.org/10.1093/nar/gku917)

This was the first project which I got involved in after I joined BGI when I was a junior student, in which I engaged in the following parts:

● Screen out candidate mitochondrial reads

● Alignment and validation of mitogenome sequences

● Mitochondrial genome annotation and data submission


